modules/TwitterStreamingImporter/src/main/resources/totetmatt/gephi/twitter/Bundle.properties

```diff
@@ -0,0 +1,46 @@
+# To change this license header, choose License Headers in Project Properties.
+# To change this template file, choose Tools | Templates
+# and open the template in the editor.
+
+CTL_SimpleAction=Twitter Stream
+CTL_SimpleTopComponent=Twitter Stream
+HINT_SimpleTopComponent=WHEEEEEEE
```

Comment:
That's what appears in the tooltip in the component and the copy can be improved I think :)